css/components/modal/_modal-footer.scss

```diff
@@ -0,0 +1,23 @@
+.modal-footer {
```

Comment:
Related concepts should stay in the same file in this case. For example a modal header or footer are related to a modal. They don't make sense on their own. If something doesn't make sense on its own, it means that it belongs somewhere else. That's a good rule of thumb.